maps/_map_override.dm

```diff
@@ -42,5 +42,9 @@
 		#undef MAP_OVERRIDE
 		#include "tgstation.dm"
 		#define MAP_OVERRIDE 7
+	#elif MAP_OVERRIDE == 8
```

Comment:
can you also push the related entry in __compile_options?

Reply:
Sure.